yaml/boundaries.yaml

```diff
@@ -1,75 +1,79 @@
 filters:
+
+  # osm
   - filter:
       admin_level: ['2', '4']
       boundary: administrative
       'tags->boundary:type': aboriginal_lands
     min_zoom: 8
-    output: {kind: aboriginal_lands, admin_level: {col: admin_level}}
+    output: {kind: aboriginal_lands, kind_detail: '3'}
```

Comment:
Updated to use the other naming scheme for aboriginal lands, namely `boundary=aboriginal_lands` in 17c3774.